data/templates/esx-boot-cfg

```diff
@@ -0,0 +1,7 @@
+bootstate=0
+title=Loading ESXi installer
+kernel=<%=tbootFile%>
+kernelopt=runweasel formatwithmbr com1_baud=115200 com1_Port=<%=comportaddress%> tty2Port=<%=comport%> debugLogToSerial=1 logPort=<%=comport%> ks=http://<%=server%>:<%=port%>/api/common/templates/esx-ks
+modules=<%=moduleFiles%>
+build=
```

Comment:
Thanks for the details Felix.  The code can be removed and replaced with a "prefix=<%=repo%>" in the boot.cfg file.  Using the prefix would let you skip the step where you are forcibly changing the user's repo input to lowercase in the code above.  Changing the case would affect URLs with auth tokens encoded in them.  I'll make a note of that in the other PR.